slicedimage/backends/_caching.py

```diff
@@ -1,31 +1,37 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import os
+import io
+
+from diskcache import Cache
 
 from ._base import Backend
 
+size_limit = 5e9
+
 
 class CachingBackend(Backend):
```

Comment:
Consider re-naming to "CacheRulesEverythingAroundMeBackend" or C.R.E.A.M.Backend? https://www.youtube.com/watch?v=PBwAxmrE194

Reply:
Best comment.